src/values.rs

```diff
@@ -96,14 +96,14 @@ impl Value {
         Self::Tuple { vs: vec![] }
     }
 
-    /// Constant Sum over units, used as predicates.
-    pub fn simple_predicate(tag: usize) -> Self {
+    /// Constant Sum over units, used as branching values.
```

Comment:
```suggestion
    /// Constant Sum of a unit value, used to control branches.
```
(Other variants could be non-unit, since we are checkable not synthesizable here)